Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -1680,8 +1680,23 @@ void Viewer_impl::sendSnapshotToClipboard(Viewer *viewer)
   QImage * snap = takeSnapshot(viewer, 95, 1, 2*viewer->size(), 1, true);
   if(snap)
   {
-    QApplication::clipboard()->setImage(*snap);
-    delete snap;
+#if defined(BOOST_MSVC)
+QApplication::clipboard()->setImage(*snap);
+QMimeData *mimeData = new QMimeData();
+QByteArray ba;
+QBuffer buffer(&ba);
+buffer.open(QIODevice::WriteOnly);
+snap->save(&buffer, "PNG"); // writes image into ba in PNG format
+buffer.close();
+mimeData->setData("PNG", ba);
+According to the doc, the ownership of mime_data is transferred to
+clipboard, so this is not a memory leak.
+QApplication::clipboard()->setMimeData(mimeData);
```

Comment:
Lines 1692-1693 cannot compile! There should be a comment.

And please fix the indentation.